RecordBuffers/src/main/java/datamine/storage/recordbuffers/ReadOnlyRecord.java

```diff
@@ -133,84 +147,105 @@ public Object getValue(T col) {
 		
 		// never return NULL
 		if (result == null) { 
-			return col.getField().getDefaultValue();
+			return field.getDefaultValue();
 		} else {
 			return result;
 		}
 	}
 	
-	public boolean getBool(T col) {
-		int offset = getOffset(col);
+	public boolean getBool(Field field) {
+		int offset = getOffset(field);
 		if (offset > 0) {
 			return buffer.getByteBuffer().get(offset) == 1 ? true : false;
 		} else {
-			return (Boolean) col.getField().getDefaultValue();
+			return (Boolean) field.getDefaultValue();
 		}
 	}
 
+	public boolean getBool(T col) {
+		return getBool(col.getField());
+	}
 	
-	public byte getByte(T col) {
-		int offset = getOffset(col);
+	public byte getByte(Field field) {
+		int offset = getOffset(field);
 		if (offset > 0) {
 			return buffer.getByteBuffer().get(offset);
 		} else {
-			return (Byte) col.getField().getDefaultValue();
+			return (Byte) field.getDefaultValue();
```

Comment:
It is possible, so it is not very recommended to use this function directly. Otherwise some kind of checking must be enforced, from the outside.